packages/polling-controller/src/StaticIntervalPollingController.test.ts

```diff
@@ -0,0 +1,242 @@
+import { ControllerMessenger } from '@metamask/base-controller';
+import { useFakeTimers } from 'sinon';
+
+import { advanceTime } from '../../../tests/helpers';
+import { StaticIntervalPollingController } from './StaticIntervalPollingController';
+
+const TICK_TIME = 1000;
+
+const createExecutePollMock = () => {
+  const executePollMock = jest.fn().mockImplementation(async () => {
+    return true;
+  });
+  return executePollMock;
+};
+
+class MyGasFeeController extends StaticIntervalPollingController<
+  any,
+  any,
+  any
+> {
+  _executePoll = createExecutePollMock();
+
+  _intervalLength = TICK_TIME;
```

Comment:
There is no property called `_intervalLength`, so this has no effect. You can test this by changing `TICK_TIME` to something else like 500. Thoughts on overriding `getIntervalLength` instead?

Reply:
yeah whoops this got left in by mistake. Why not just call the `setIntervalLength`?

Reply:
Yeah, that would work too!

Reply:
done here: https://github.com/MetaMask/core/pull/3636/commits/823aaa188f572a8aa76973a012ae9bfa8da518e0